README.md

```diff
@@ -28,13 +28,15 @@ repo_mirror_packages:
   - logrotate
   - git
   - systemd
+  - yarnpkg
+  - nodejs
+  - make
 ```
 
 ```yaml
+# defaults/main.yml
 ---
```

Comment:
The `---` indicates the start of the YAML document, of which the comment is a part of:
```suggestion
---
# defaults/main.yml
```